addons/l10n_es/upgrades/5.3/post-migrate.py

```diff
@@ -0,0 +1,7 @@
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    for company in env['res.company'].search([('chart_template', 'in', ('es_assec', 'es_common', 'es_coop_full', 'es_coop_pymes', 'es_full', 'es_pymes'))]):
+        env['account.chart.template'].try_loading('dk', company)
```

Comment:
wrong code

Reply:
and the path should be `addons/l10n_es/migrations/5.3/post-migrate.py`

Reply:
:facepalm: you didn't see anything :eyes:

Reply:
saw nothing :walking_woman:  but you can leave the old file in the old directory

Reply:
I think we should use an `end-migrate` script instead. (Although it may not cause issues now it may cause an issue in the future and may be a good idea in general to rather use `end` scripts). (I also quickly checked the issue with oco)
In `l10n_uk` there is a migration error due to using the `post-migrate` instead of the `end-migrate` (PR linked below). The issue is that the `l10n_uk_reports` is not loaded at the time of the `post-migrate`.
The `l10n_uk_reports` defines a custom handler (`'l10n_uk.tax.report.handler'`). The name is stored in the DB (I think). Thus during the (re)loading of the chart template we try to call this custom handler. But since `l10n_uk_reports` is not loaded yet we cannot find it. A traceback occurs.
PR from upgrade team: https://github.com/odoo/odoo/pull/181154
@jjmaksoud What do you think?

Reply:
Yes @svfu-odoo has a point, i agree with moving to `end`. my original comment was on the folder name only.